src/providers/diagnostics.ts

```diff
@@ -46,7 +47,7 @@ export function findStaleConfigs(
             createDiagnostic(DiagnosticCode.staleCheck, {
               range,
               configName,
-              reason: '0 checks in the past 30 days',
+              reason: staleCheck[0].reason,
```

Comment:
why does this return an array if you only use the first value?  Can this just return a single value?

Reply:
It's the associated project. Since a config could be identified as stale in multiple projects, I just defaulted to the first one when listing the reason.